migration-guide/src/submit-api.md

```diff
@@ -0,0 +1,29 @@
+# submit-api
+
+GraphQL doesn't support submission of data, so the following endpoints
+are only provided by Rosetta.
+
+## Transactions
+
+#### [/api/submit/tx](https://input-output-hk.github.io/cardano-rest/submit-api/#operation/postTransaction)
+
+Submit an already serialized transaction to the network.
+
+=== "cardano-rest"
+
+    Assuming `data` is a serialized transaction on the file-system.
+    ```console
+    $ curl -X POST --header "Content-Type: application/cbor" --data-binary @data http://localhost:8101/api/submit/tx
+    ```
+
+    <details>
+      <summary>see JSON response</summary>
+    ```json
+    92bcd06b25dfbd89b578d536b4d3b7dd269b7c2aa206ed518012cffe0444d67f
+    ```
+    </details>
+
+=== "cardano-rosetta"
+
+    This can be done via the [/construction/submit](https://www.rosetta-api.org/docs/ConstructionApi.html#constructionsubmit) endpoint.
```

Comment:
So... the rosetta documentation says:

```
{
    "network_identifier": {
        "blockchain": "bitcoin",
        "network": "mainnet",
        "sub_network_identifier": {
            "network": "shard 1",
            "metadata": {
                "producer": "0x52bc44d5378309ee2abf1539bf71de1b7d7be3b5"
            }
        }
    },
    "signed_transaction": "string"
}
```

So the payload for the transactions is a `string`. That doesn't seem it will accept `application/cbor`... so what's the actual format here?

Reply:
https://github.com/input-output-hk/cardano-rosetta/blob/7769bdfecc8449bb1ba3f5001754ab663ae700ba/cardano-rosetta-server/src/server/controllers/construction-controller.ts#L125

https://github.com/input-output-hk/cardano-rosetta/blob/7769bdfecc8449bb1ba3f5001754ab663ae700ba/cardano-rosetta-server/src/server/utils/data-mapper.ts#L326